tests/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/ILikeTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Tests\Doctrine\ORM\Query\AST\Functions;
+
+use MartinGeorgiev\Doctrine\ORM\Query\AST\Functions\ILike;
+use MartinGeorgiev\Tests\Doctrine\Fixtures\Entity\ContainsText;
+
+class ILikeTest extends TestCase
+{
+    protected function getStringFunctions(): array
+    {
+        return [
+            'ILIKE' => ILike::class,
+        ];
+    }
+
+    protected function getExpectedSqlStatements(): array
+    {
+        return [
+            "SELECT c0_.text ilike 'TEST' AS sclr_0 FROM ContainsText c0_",
```

Comment:
This use-case is a bit unusual to me but is completely fine. Thanks for adding some tests!